dist/jsdoc.def.js

```diff
@@ -19,7 +19,7 @@
  * @property {Object} alias - Rename default queries and mutations with alias. Example: { count:'myCountQuery', fetch: 'myQuery', create: 'myCreateMutation', destroy: 'myDeleteMutation, update: 'myUpdateMutation' }
  * @property {Array<string>} excludeQueries - Exclude default queries. Example: [ 'count', 'fetch' ]
  * @property {Array<string>} excludeMutations - Exclude default mutations. Example: [ 'create', 'update', 'destroy' ]
- * @property {Array<string>} excludeSubscriptions - Exclude default subscriptions. Example: [ 'default', 'create', 'update', 'destroy' ]
+ * @property {Array<string>} excludeSubscriptions - Exclude default subscriptions. Example: [ 'create', 'update', 'destroy' ]
```

Comment:
Why you removed it @emanuelet ? it's currently used by subscriptions filter @alirizwan

Reply:
I haven't removed anything. I simply runned a `npm run build`

Reply:
ok sorry, my mistake